react/features/base/config/configWhitelist.ts

```diff
@@ -81,6 +81,11 @@ export default [
     'brandingRoomAlias',
     'debug',
     'debugAudioLevels',
+    'deeplinking.disabled',
+    'deeplinking.showImage',
+    'deeplinking.desktop.appName',
+    'deeplinking.ios.appName',
```

Comment:
I don't think the mobile app names were overridable before, were they?

Reply:
they where using the NATIVE_APP_NAME which was whitelisted- but yeah it might make sense to remove from whitelist.

Reply:
shall I remove? :)

Reply:
Removed.